Frameworks/PyTorch/Getting-Started-Samples/PyTorch_HelloWorld/PyTorch_Hello_World.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+'''
+==============================================================
+ Copyright © 2019 Intel Corporation
+
+ SPDX-License-Identifier: MIT
+==============================================================
+'''
+
+import torch
+import torch.nn as nn
+from torch.utils import mkldnn
+from torch.utils.data import Dataset, DataLoader
+
+'''
+BS_TRAIN: Batch size for training data
+BS_TEST:  Batch size for testing data
+EPOCHNUM: Number of epoch for training
+'''
+BS_TRAIN = 50
+BS_TEST  = 10
+EPOCHNUM = 2
+
+'''
+TestDataset class is inherited from torch.utils.data.Dataset.
+Since data for training involves data and ground truth, a flag "train" is defined in the initialization function. When train is True, instance of TestDataset gets a pair of training data and label data. When it is False, the instance gets data only for inference. Value of the flag "train" is set in __init__ function.
+In __getitem__ function, data at index position is supposed to be returned.
+__len__ function returns the overall length of the dataset.
+'''
+class TestDataset(Dataset):
+    def __init__(self, train = True):
+        super(TestDataset, self).__init__()
```

Comment:
I guess this is nowadays
```suggestion
        super().__init__()
```

Reply:
Hi, since PyTorch codes also use this style, we follow this at this moment.
https://github.com/pytorch/pytorch/search?q=__init__%28%29&unscoped_q=__init__%28%29